client/me/purchases/purchase-item/index.jsx

```diff
@@ -137,138 +159,145 @@ class PurchaseItem extends Component {
 		return null;
 	}
 
-	placeholder() {
-		return (
-			<span className="purchase-item__wrapper">
-				<div className="purchase-item__plan-icon" />
-				<div className="purchase-item__details">
-					<div className="purchase-item__title" />
-					<div className="purchase-item__purchase-type" />
-					<div className="purchase-item__purchase-date" />
-				</div>
-			</span>
-		);
-	}
+	getPurchaseType() {
+		const { purchase, site, translate, slug, showSite } = this.props;
+
+		if ( showSite ) {
+			return translate( '%(purchaseType)s for {{button}}%(site)s{{/button}}', {
+				args: {
+					purchaseType: purchaseType( purchase ),
+					site: site.name,
+				},
+				components: {
+					button: (
+						<button
+							className="purchase-item__site-name"
+							onClick={ ( event ) => {
+								event.stopPropagation();
+								event.preventDefault();
+								page( getPurchaseListUrlFor( slug ) );
+							} }
+							title={ translate( 'View subscriptions for %(siteName)s', {
+								args: {
+									siteName: site.name,
+								},
+							} ) }
+						/>
+					),
+				},
+			} );
+		}
 
-	scrollToTop() {
-		window.scrollTo( 0, 0 );
+		return purchaseType( purchase );
 	}
 
-	renderIcon() {
+	getPaymentMethod() {
 		const { purchase } = this.props;
 
-		if ( ! purchase ) {
-			return null;
-		}
+		if ( isRenewing( purchase ) ) {
+			const imgSrc =
+				purchase.payment.type === 'credit_card'
+					? getPaymentMethodImageURL( purchase.payment.creditCard.type )
+					: payPalImage;
+			const altText =
+				purchase.payment.type === 'credit_card'
+					? purchase.payment.creditCard.type
+					: purchase.payment.typ;
```

Comment:
Looks like a typo here.

Reply:
Thanks for catching that!

Reply:
Fixed in this commit: https://github.com/Automattic/wp-calypso/pull/47610/commits/c5e06c98ad12478b46d5e9ad8e660d40a428e02c